data/json/vehicleparts/combustion.json

```diff
@@ -36,6 +36,15 @@
     "//": "30% energy efficiency",
     "description": "A combustion engine.  Burns gasoline fuel from a tank in the vehicle."
   },
+  {
+    "abstract": "gas_turbine_engine",
+    "copy-from": "engine_combustion",
+    "type": "vehicle_part",
+    "fuel_type": "gasoline",
+    "m2c": 60,
+    "//": "25% energy efficiency",
+    "description": "An advanced combustion engine.  Burns gasoline fuel from a tank in the vehicle. Better power-to-weight ratio than a traditional reciprocating engine, but consumes more fuel."
```

Comment:
Most turbines are multifuel.  The Abrams tank engine famously is, for instance.

Reply:
How multi-fuel do you want to go? I've changed the engines over to also accepting diesel and lamp oil, though left the spawning vehicle tanks filled with gas. Looks like it could run on methanol and ethanol as well but the fuel consumption would be be crazy.

Reply:
Up to you.  Diesel engines run on methanol and ethanol, but each option adds another line to the engine control menu for each engine.